sphinx/templates/latex/tabular.tex_t

```diff
@@ -1,3 +1,11 @@
+<%- if table.align -%>
+\begin{table}[htb]
+  <%- if table.align == 'center' -%>
+    \centering
+  <%- else -%>
+    \begin{flush<%= table.align%>}
+  <%- endif %>
+<% endif -%>
 <%- if table.caption -%>
 \begin{threeparttable}
```

Comment:
In the end I changed my mind about the last point, because I agree the simple thing to do is remove threeparttable use completely and use always table (except for longtable) and so we naturally must allow user config to be `H` all the time, if only because this is how to best emulate current Sphinx behaviour.